changelog.d/798.docker

```diff
@@ -0,0 +1 @@
+Update Dockerfile to allow building for ARM arches.
```

Comment:
I see why you've done this, but I worry this might lead people to think we build ARM images too..which we don't yet. Could you maybe word it to sound like we've unstuck a dependency?

Reply:
Done